portfolio/src/main/webapp/script.js

```diff
@@ -27,3 +27,15 @@ function nextImage() {
   imageContainer.src = currentImage;
   
 }
+
+/**
+ * Fetches data and adds to html
+ */
+function getHelloNameData() {
+    fetch('/data').then(response => response.json()).then((hello) => {
+        const msgContainer = document.getElementById('hello-container');
+
+        msgContainer.innerHTML = hello[0] + " " + 
```

Comment:
Lets use a for each here that way we can dynamically change messages and it will update